scss/_variables.scss

```diff
@@ -1080,6 +1080,8 @@ $nav-pills-link-active-bg:          $component-active-bg !default;
 // Navbar
 
 // scss-docs-start navbar-variables
+$navbar-breakpoints:                $grid-breakpoints !default
```

Comment:
This change should now go in the `_maps.scss` style sheet as it’s a map that reuses an existing one.